src/test/java/emissary/util/LineTokenizerTest.java

```diff
@@ -1,104 +1,105 @@
     }
 
     @Test
-    public void testParsing() {
+    void testParsing() {
         LineTokenizer lt = new LineTokenizer("ABC\nDEF\r\nGHI\n\nJKL\r\n\r\nMNO".getBytes());
-        assertEquals("Line counting", 7, lt.countTokens());
-        assertTrue("Token setup", lt.hasMoreTokens());
-        assertEquals("Index should be at the beginning of the string", 0, lt.index);
-        assertEquals("Line parsing with LF", "ABC", lt.nextToken());
-        assertEquals("Index should be at the index of 'D'", 4, lt.index);
-        assertEquals("Line parsing with CRLF", "DEF\r", lt.nextToken());
-        assertEquals("Index should be at the index of 'G'", 9, lt.index);
-        assertEquals("Line parsing before double LF", "GHI", lt.nextToken());
-        assertEquals("Index should be at the index of the '\n' before 'JKL'", 13, lt.index);
-        assertEquals("Blank line with LF", "", lt.nextToken());
-        assertEquals("Index should be at the index of 'J'", 14, lt.index);
-        assertEquals("Line pargin before double CRLF", "JKL\r", lt.nextToken());
-        assertEquals("Index should be at the index of '\r' before '\nMNO'", 19, lt.index);
-        assertEquals("Blank line with CRLF", "\r", lt.nextToken());
-        assertEquals("Index should at the index of 'M'", 21, lt.index);
-        assertEquals("Trailing portion without LF", "MNO", lt.nextToken());
-        assertEquals("Index should be at the end of the string", 24, lt.index);
-        assertEquals("No remaining tokens", null, lt.nextToken());
-        assertEquals("Index should not have changed", 24, lt.index);
+        assertEquals(7, lt.countTokens(), "Line counting");
+        assertTrue(lt.hasMoreTokens(), "Token setup");
+        assertEquals(0, lt.index, "Index should be at the beginning of the string");
+        assertEquals("ABC", lt.nextToken(), "Line parsing with LF");
+        assertEquals(4, lt.index, "Index should be at the index of 'D'");
+        assertEquals("DEF\r", lt.nextToken(), "Line parsing with CRLF");
+        assertEquals(9, lt.index, "Index should be at the index of 'G'");
+        assertEquals("GHI", lt.nextToken(), "Line parsing before double LF");
+        assertEquals(13, lt.index, "Index should be at the index of the '\n' before 'JKL'");
+        assertEquals("", lt.nextToken(), "Blank line with LF");
+        assertEquals(14, lt.index, "Index should be at the index of 'J'");
+        assertEquals("JKL\r", lt.nextToken(), "Line pargin before double CRLF");
+        assertEquals(19, lt.index, "Index should be at the index of '\r' before '\nMNO'");
+        assertEquals("\r", lt.nextToken(), "Blank line with CRLF");
+        assertEquals(21, lt.index, "Index should at the index of 'M'");
+        assertEquals("MNO", lt.nextToken(), "Trailing portion without LF");
+        assertEquals(24, lt.index, "Index should be at the end of the string");
+        assertNull(lt.nextToken(), "No remaining tokens");
```

Comment:
I know it's not yours, but this messages seems to indicate a success rather than a failure.